v3/newrelic/config.go

```diff
@@ -725,6 +725,16 @@ func (c Config) maxTxnEvents() int {
 	return configured
 }
 
+// maxSpanEvents returns the configured maximum number of Span Events if it has been configured
+// and is less than the default maximum; otherwise it returns the default max.
+func (c Config) maxSpanEvents() int {
+	configured := c.DistributedTracer.ReservoirLimit
+	if configured < 0 || configured > internal.MaxSpanEvents {
```

Comment:
you may want to confirm this in agent-dev, but I think the expectation is that agents no longer limit data capacity requests. The connect server will do it in the connect response.

Reply:
Good idea, I'll verify this and update the PR if necessary